docs/GettingStarted.md

```diff
@@ -1 +1,25 @@
-# Getting Started
\ No newline at end of file
+# Getting Started
```

Comment:
Could maybe add a list of the components to this page with links to the individual component docs?